digital_image_processing/change_brightness.py

```diff
@@ -0,0 +1,23 @@
+from PIL import Image
+
+def change_brightness(img: Image, level: float) -> Image:
+    """
+    Function to change brightness
+    """
+    factor = (259 + (level + 255)) / (255 + (259 - level))
+
+    def brightness(c: int) -> float:
+        """
+        Fundamental Transformation/Operation that'll be performed on
+        every bit.
+        """
+        return 128 + factor * (c - 128)
+
+    return img.point(brightness)
+
+if __name__ == "__main__":
+    # Load image
+    with Image.open("image_data/lena.jpg") as img:
+        # Change brightness to 170
+        cont_img = change_brightness(img, 300)
```

Comment:
How does 170 relate to 300?!?  If the comment is saying exactly what the code is saying then delete the comment.